brave/brave-core
# Connected status

Resolves https://github.com/brave/brave-browser/issues/5631

UI PR: https://github.com/brave/brave-ui/pull/526
UI PR: https://github.com/brave/brave-ui/pull/527

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards (staging)
- visit `3zsistemi.si` or any other side that has connected state
- open the panel
- make sure that publisher has checkmark and at the same time not verified text
- click on tip
- make sure that you see not verified text in the banner
- tip
- make sure that contribution went into pending contributions list 

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
